apps/wallet/src/ui/app/pages/home/transfer-coin/SendTokenForm.tsx

```diff
@@ -107,41 +113,58 @@ export function SendTokenForm({
 }: SendTokenFormProps) {
     const activeAddress = useActiveAddress();
     // Get all coins of the type
-    const { data: coinsData, isLoading: coinsIsLoading } = useGetCoins(
-        coinType,
-        activeAddress!
-    );
 
-    const { data: suiCoinsData, isLoading: suiCoinsIsLoading } = useGetCoins(
-        SUI_TYPE_ARG,
-        activeAddress!
-    );
+    const {
+        data: coinsData,
+        isLoading: coinsIsLoading,
+        hasNextPage: coinsHasNextPage,
+        fetchNextPage: fetchNextPageCoins,
+        isFetchingNextPage: isFetchingNextPageCoin,
+    } = useGetCoins(coinType, activeAddress!, MAX_COINS_PER_REQUEST);
```

Comment:
Maybe we should create another hook for the wallet to fetch all coins something like `useGetAllCoins` (should be ok to move it to core as well). We could reuse the new `useGetCoins` but I think I would prefer the hook we already had. This approach feels a bit hacky

Reply:
Reply

Reply:
Leaving it in the wallet hooks since the call is relatively expensive